routers/web/repo/view.go

```diff
@@ -503,8 +508,12 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 			ctx.Data["IsMarkup"] = true
 			ctx.Data["MarkupType"] = markupType
 			var result strings.Builder
+			if !detected {
+				markupType = ""
+			}
```

Comment:
I'm confused by this paragraph.
According to my rewrite, you would need to switch it around:
```suggestion
			if detected {
				markupType = ""
			}
```
But why is it there? It doesn't make any sense to me.
That either indicates that your logic is currently wrong, or that I'm making a conclusion error somewhere, which would be a bit surprising.

Reply:
to not pass it down to RenderContext Type field

Reply:
Yeah, it makes sense now, but I would still like either the inverted meaning of the boolean, or at least a comment why the other way around was chosen.

Reply:
I added comment in code about this